EFCore.CheckConstraints/CheckConstraints/Internal/CheckConstraintsConventionSetPlugin.cs

```diff
@@ -2,21 +2,35 @@
 using Microsoft.EntityFrameworkCore.Metadata.Conventions;
 using Microsoft.EntityFrameworkCore.Metadata.Conventions.Infrastructure;
 using JetBrains.Annotations;
+using Microsoft.EntityFrameworkCore.Storage;
 
 namespace EFCore.CheckConstraints.Internal
 {
     public class CheckConstraintsConventionSetPlugin : IConventionSetPlugin
     {
         readonly IDbContextOptions _options;
-        public CheckConstraintsConventionSetPlugin([NotNull] IDbContextOptions options) => _options = options;
+        readonly ISqlGenerationHelper _sqlGenerationHelper;
```

Comment:
I see inconsistency in modifiers.

Reply:
It is consistent, just not the EF Core way :trollface: 

But to not hurt team members' eyes, I will copy the EF Core .editorconfig and Jetbrains settings and align to that.